Scripts: Disable markup in error messages

Unrud opened the pull request:
Error messages are displayed using the following methods:

* **zenity** parses pango markup and replaces escaped characters
* **kdialog** replaces (some) escaped characters
* **gtk-dialog.py** replaces `\n`
* **printf** interprets format strings and replaces escaped characters
* **echo** just displays the text

The error messages themself contain escaped characters and paths from variables.

This PR unifies the behavior by:

* Use **printf** to format error messages and replace escaped characters
* Setting `--no-markup` for **zenity** to disable pango markup and escaped characters
* Remove `\n` replacement from **gtk-dialog.py**.
* Use plain **echo** instead of **printf**

Unrud committed:
Scripts: Disable markup in error messages

Error messages are displayed using the following methods:

* **zenity** parses pango markup and replaces escaped characters
* **kdialog** replaces (some) escaped characters
* **gtk-dialog.py** replaces `\n`
* **printf** interprets format strings and replaces escaped characters
* **echo** just displays the text

The error messages themself contain escaped characters and paths from variables.

This PR unifies the behavior by:

* Use **printf** to format error messages and replace escaped characters
* Setting `--no-markup` for **zenity** to disable pango markup and escaped characters
* Remove `\n` replacement from **gtk-dialog.py**.
* Use plain **echo** instead of **printf**

anvilvapre commented on packaging/linux/gtk-dialog.py:
```diff
@@ -43,6 +43,6 @@ def __init__(self, title, text):
     parser.add_argument('--text', type=str, required=False, default='')
     args = parser.parse_args()
     if args.type == 'question':
-        Question(args.title, args.text.replace('\\n', '\n'))
```
I assume this had no effect to begin with? Also in the old situation there would be no dobule backslashes to replace?

Unrud replied:
I think you are confusing it with single quoted strings in shell. In Python `"` and `'` behave the same and support escape characters.

anvilvapre replied:
no, i meant, even in the old situation the message text would not have had two slashes in them. also in the past the newlines were newline characters. (?)

Unrud replied:
I'm not following.
In Python `'\\n'` becomes `\n` and `'\n'` becomes `<newline>`.
In Shell scripts `"\n"` stays `\n`. `printf "\n"` outputs `<newline>` and `echo "\n"` outputs `\n`.

anvilvapre replied:
you are right. i made the wrong assumption about the last line.

Unrud committed:
Use python3 for gtk-dialog.py

As mentioned in https://github.com/OpenRA/OpenRA/pull/20059#issuecomment-1166288560

abcdefg30 commented:
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/5701cce1441309e0fff64c270ed1075c86b5e96b...c4eb72ecec6e0a03d4b85eb345b6052d3c7d1833)
